src/redux/books/books.js

```diff
@@ -0,0 +1,18 @@
+const ADDING = 'ADDING';
+const REMOVING = 'REMOVING';
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md) for this project, " `Write your book's actions and reducer (in /src/redux/books/books.js) using the ducks pattern` ", please kindly define your actions using the [ducks pattern](https://github.com/erikras/ducks-modular-redux) like so: 
```
const ADD_BOOK = bookstore/books/ADD_BOOK 
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```